### PR TITLE
Improve document for developers: Windows CGO, unit test option

### PR DESCRIPTION
**Pull request by wxiaoguang**
1. Clarify how to compile Gitea on Windows with CGO_ENABLED
2. Document a useful environment variable GITEA_UNIT_TESTS_VERBOSE

### CODE REVIEW

File: docs/content/doc/developers/hacking-on-gitea.en-us.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ One of these three distributions of Make will run on Windows:
   - MSYS2 is a collection of tools and libraries providing you with an easy-to-use environment for building, installing and running native Windows software, it includes MinGW-w64. 
   - In MingGW-w64, the binary is called `mingw32-make.exe` instead of `make.exe`. Add the `bin` folder to `PATH`.
   - In MSYS2, you can use `make` directly. See [MSYS2 Porting](https://www.msys2.org/wiki/Porting/).
+  - To compile Gitea with CGO_ENABLED (eg: SQLite3), you must use [tdm-gcc](https://jmeubank.github.io/tdm-gcc/) instead of MSYS2 gcc, because MSYS2 gcc headers lack some Windows-only CRT functions like `_beginthread`.
```
Comment (lunny) [inline docs/content/doc/developers/hacking-on-gitea.en-us.md]: I think `tdm-gcc` is a recommend but not a must.

**Review by lunny — APPROVED** [review]
Otherwise LGTM

File: docs/content/doc/developers/hacking-on-gitea.en-us.md (hunk 1 above)
Comment (wxiaoguang) [reply docs/content/doc/developers/hacking-on-gitea.en-us.md]: OK, updated, `you might need to use [tdm-gcc]`

**Review by 6543 — APPROVED** [review]

**Comment by wxiaoguang** [comment]
trigger bot

**Pull request merged**